### PR TITLE
Add integration with Wildix PBX

### PR DESCRIPTION
**Pull request by gorets**
Integration with Wildix PBXs https://www.wildix.com/

Api documentation: https://docs.wildix.com/wms/index.html

### CODE REVIEW

File: docs-v2/integrations/all/wildix.mdx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+| [Auth (OAuth)](/guides/oauth)                                                      | ✅                              |
+| [Syncs](/guides/sync) & [Actions](/guides/actions)                                                              | ✅                              |
+| [Nango Proxy](/guides/proxy)                          | ✅   |
+| Auto-pagination                     | 🚫 (time to contribute: &lt;1h) |
+| API-specific rate limits | 🚫 (time to contribute: &lt;1h) |
+
+<Tip>We can implement missing features in &lt;48h, just ask for it in the [community](https://nango.dev/slack).</Tip>
+## Getting started
+
+-   [How to register an Application](https://api.accelo.com/docs/#registering-your-application)
```
Comment (bastienbeurier) [inline docs-v2/integrations/all/wildix.mdx]: These links are still from Accelo

File: docs-v2/integrations/all/wildix.mdx [hunk 2]
```diff
@@ -0,0 +1,43 @@
+---
```
Comment (bastienbeurier) [inline docs-v2/integrations/all/wildix.mdx]: Also, this page needs to be referenced in the `mint.json` file.

**Review by bastienbeurier — CHANGES_REQUESTED** [review]
Super cool addition! There are some minor changes to make and it's ready to merge.

File: docs-v2/integrations/all/wildix.mdx (hunk 1 above)
Comment (gorets) [reply docs-v2/integrations/all/wildix.mdx]: Thanks, fixed

File: docs-v2/integrations/all/wildix.mdx (hunk 2 above)
Comment (gorets) [reply docs-v2/integrations/all/wildix.mdx]: Added this page to all list and link to integration-templates/wildix-pbx

File: docs-v2/integrations/all/wildix.mdx (hunk 1 above)
Comment (bastienbeurier) [reply docs-v2/integrations/all/wildix.mdx]: Weirdly I still see some accelo links in the latest code!

File: docs-v2/integrations/all/wildix.mdx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+| [Nango Proxy](/guides/proxy)                          | ✅   |
+| Auto-pagination                     | 🚫 (time to contribute: &lt;1h) |
+| API-specific rate limits | 🚫 (time to contribute: &lt;1h) |
+
+<Tip>We can implement missing features in &lt;48h, just ask for it in the [community](https://nango.dev/slack).</Tip>
+## Getting started
+
+-   [How to register an Application](https://wildix.atlassian.net/wiki/x/3B-OAQ)
+-   [Api documentation for Wildix PBX](https://docs.wildix.com/wms/index.html)
+-   [OAuth-related docs](https://api.accelo.com/docs/#authentication)
```
Comment (bastienbeurier) [inline docs-v2/integrations/all/wildix.mdx]: Still seeing 3 accelo links

File: docs-v2/mint.json [hunk 4]
```diff
@@ -93,6 +93,7 @@
                         "integration-templates/notion",
                         "integration-templates/salesforce",
                         "integration-templates/slack",
+                        "integration-templates/wildix-pbx",
```
Comment (bastienbeurier) [inline docs-v2/mint.json]: I don't see the corresponding `.mdx` file for this.

**Review by bastienbeurier — CHANGES_REQUESTED** [review]

File: docs-v2/mint.json (hunk 4 above)
Comment (gorets) [reply docs-v2/mint.json]: Sorry, I saved it with other filename. Fixed.

File: docs-v2/integrations/all/wildix.mdx (hunk 3 above)
Comment (gorets) [reply docs-v2/integrations/all/wildix.mdx]: fixed

File: docs-v2/integrations/all/wildix-pbx.mdx [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+-   [How to register an Application](https://wildix.atlassian.net/wiki/x/3B-OAQ)
+-   [Api documentation for Wildix PBX](https://docs.wildix.com/wms/index.html)
+-   [OAuth-related docs](https://docs.wildix.com/wms/index.html#section/Authentication)
+-   [List of OAuth scopes](https://docs.wildix.com/wms/index.html#section/Authentication)
+
+<Tip>Need help getting started? Get help in the [community](https://nango.dev/slack).</Tip>
+
+## Connection configuration in Nango
+
+Wildix PBX [requires a user specific subdomain](https://api.accelo.com/docs/#oauth2-0-uri) to run OAuth.
```
Comment (bastienbeurier) [inline docs-v2/integrations/all/wildix-pbx.mdx]: Still an `accelo` link here :D

File: docs-v2/mint.json (hunk 4 above)
Comment (bastienbeurier) [reply docs-v2/mint.json]: If you add this line, you must add a dedicated docs page under `docs-v2/integration-templates/wildix-pbx.mdx`. But we're deprecating that folder, so it's best to just remove that line if you don't mind.

File: packages/shared/flows.yaml [hunk 6]
```diff
@@ -554,6 +554,20 @@ integrations:
         channel_id: string
         user_id: string
         reaction_name: string
+  wildix-pbx:
```
Comment (bastienbeurier) [inline packages/shared/flows.yaml]: Good catch, but this is automatically generated, so it's better to remove this change!

**Review by bastienbeurier — CHANGES_REQUESTED** [review]

File: docs-v2/mint.json (hunk 4 above)
Comment (gorets) [reply docs-v2/mint.json]: Removed

File: docs-v2/integrations/all/wildix-pbx.mdx (hunk 5 above)
Comment (gorets) [reply docs-v2/integrations/all/wildix-pbx.mdx]: wtf... changed :(

File: packages/shared/flows.yaml (hunk 6 above)
Comment (gorets) [reply packages/shared/flows.yaml]: Removed

**Review by bastienbeurier — APPROVED** [review]

**Pull request merged**